lib/datastore/transaction.js

```diff
@@ -42,6 +42,11 @@ var extend = require('extend');
  *     connection to Google Cloud Datastore.
  * @param {string} projectId - Dataset ID. This is your project ID from the
  *     Google Developers Console.
+ *
+ * @example
+ * var Transaction = require('gcloud/lib/datastore/transaction');
```

Comment:
Yeah, dev docs are hidden because of the `/*!` header. Can you add a comment / line in the dev doc block about why that's there?